docs/intro/safety_concepts.md

```diff
@@ -0,0 +1,13 @@
+# Safety concepts
+
+EB corbos Linux for Safety Applications (EBcLfSA) is the safety-dedicated SDK built on top of EB corbos Linux.
+
+EBcLfSA allows you to build high integrity (HI) applications that can communicate securely between them, or with the rest of the low integrity (LI) system.
+
+Key concepts of EBcLfSA include the HI supervisor, the Health Monitor, and shared memory communication.
+
+You can refer to the following chapters to learn more about safety developments within EB corbos Linux/EBcLfSA:
+
+* Building a [QEMU EBcLfSA image](../examples/qemu_ebclfsa.md)
+* [EBcLfSA demo application](../apps/ebclfsa_demo.md)
+
```

Comment:
I don't think this should be added here without consulting with the safety architect first. On top of that EBcLfSA is NOT the SDK, it is the whole system